src/services/application/loginService.ts

```diff
@@ -23,9 +23,8 @@ export const loginService = async (
     } catch (e) {
       console.log(e);
 
-      return "/teams";
+      return "/error";
     }
   }
-  console.log("User does not exist");
```

Comment:
I will keep some output here so we know why its failing

Reply:
This was only for testing this specific case but it is not always because the user does not exist so we decided to delete it.